### PR TITLE
net: use m_client_interface rather than uiInterface

### PR DESCRIPTION
**Pull request by fanquake**
First commit fixes two cases where we were using `uiInterface`, rather than `clientInterface`.
Second commit renames `clientInterface` to `m_client_interface`, to match banman.

### CODE REVIEW

File: src/net.cpp [hunk 1]
```diff
@@ -1304,8 +1304,8 @@ void CConnman::NotifyNumConnectionsChanged()
     }
     if(vNodesSize != nPrevNodeCount) {
         nPrevNodeCount = vNodesSize;
-        if(clientInterface)
-            clientInterface->NotifyNumConnectionsChanged(vNodesSize);
+        if(m_client_interface)
```
Comment (MarcoFalke) [inline src/net.cpp:1307]:
```suggestion
        if (m_client_interface) {
```

Comment (fanquake) [reply src/net.cpp]: Done.

**Review by hebasto — APPROVED** [review]
ACK d8ba6327b21c755582ab3c83fc31bc6c523c41c3, verified that `uiInterface` is replaced in all sites.

**Review by Zero-1729 — COMMENTED** [review]
crACK d8ba6327b21c755582ab3c83fc31bc6c523c41c3

**Pull request merged**